### PR TITLE
Updated contact form

### PR DESCRIPTION
**Pull request by himanshujaidka**

### CODE REVIEW

File: editor.html [hunk 1]
```diff
@@ -186,5 +186,24 @@
...
+
+      <!--Input/Output windows end here-->
+    </div>
+    
+   
+
+    <!--Dynamic Footer-->
+    <script src="assets/js/jquery.js"></script>
+    <script src="assets/js/main.js"></script>
+
```
Comment (smaranjitghose) [inline editor.html:200]: Please pull the current changes to the codebase

File: index.html [hunk 2]
```diff
@@ -130,14 +130,52 @@ <h3>Open Source Programs we are a part of </h3>
                         <div class="inside-contact">
                             <h2>Get In <span>Touch</span></h2>
                             <h3><span id="confirm"></h3>
-                            <p>Name</p>
+                            <p>First Name *</p>
                             <input id="txt_name" type="text" class="contact-input" required>
-                            <p>Subject</p>
+                            <p>Last Name *</p>
```
Comment (smaranjitghose) [inline index.html:135]: We already have the required attribute using HTML.

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
The form is unresponsive. Please take care of that.

Adjust the captcha button accordingly so that on hovering over the feedback options it does not shift down ( An unpreferred UX)

**Comment by himanshujaidka** [comment]
@smaranjitghose pls have a look in it

**Pull request closed without merging**